utils/cooperations/coopsAggregateOptions.js

```diff
@@ -0,0 +1,19 @@
+const getRegex = require('../getRegex')
+
+const coopsAggregateOptions = (params = {}, query) => {

```

Comment:
why do you need params if you don't use any data from it?

Reply:
removed